### DISCUSSION TITLE
Habits on certain days of the week

### DISCUSSION THREAD

**Discussion by Milena-van-der-Molen** [iSoron/uhabits#1473] [Feature Requests]
The option to track habits that are to be done a few times a week is great, but some habits really need to be done on certain days of the week! It would be great to have a way of tracking and getting reminders for these kinds of habits. The option to connect habits to certain weekdays can be seen in several apps, but to have this option added to this app would make it the best of all. How can this be implemented?

**Comment by TeTiMa** [comment]
This would be great 😃👍

**Comment by LordJon** [comment]
Isn't this already there? When you set something to do 3 times a week it will remember on which days you've done it the first time.

But since I'm still on v1.8.12 I could be wrong that this isn't available on 2.0.x due to custom frequencies being gone (will be re-added in 2.1).

**Comment by judelhaye** [comment]
That's the only missing feature for my use case : being able to say "get the trashbin out every fridays and the recyclables trashbin every other sunday."

**Comment by schielb** [comment]
When you are creating a habit, you can set the frequency, and then you can set your **reminders** to be certain days of the week.

So if you need a reminder to take out the trash on Fridays, for example, a way you could do it is create the habit, set the 'Frequency' to one time every week, set the reminder **time** for \<whenever you need to do it on Friday\>, and then set the reminder **day** to only occur on Friday.

I'll admit, it could be more robust/explicit, but this seems to be a good way to make that work.

**Reply by judelhaye** [reply]
Ho great ! I figured it out, the day selector only appears after you set a time. Hidden feature, but golden feature. Thanks for your answer !
